libraries/chain/transaction_context.cpp

```diff
@@ -850,6 +738,7 @@ namespace sysio { namespace chain {
       flat_set<account_name> actors;
 
       bool one_auth = false;
+      auto payer = ""_n;
```

Comment:
name payer;

will be more efficient (assuming the compiler doesn't optimize it away??).